lib/url.js

```diff
@@ -827,8 +827,9 @@ Url.prototype.resolveObject = function resolveObject(relative) {
       //occasionally the auth can get stuck only in host
       //this especially happens in cases like
       //url.resolveObject('mailto:local1@domain1', 'local2@domain2')
-      const authInHost = result.host && result.host.indexOf('@') > 0 ?
-                       result.host.split('@') : false;
+      const authInHost =
+        result.host && result.host.indexOf('@') > 0 ?
+          result.host.split('@') : false;
```

Comment:
This doesn’t need to be a ternary anyway, right? Just `result.host && result.host.indexOf('@') > 0 && result.host.split('@')` works as well?